tests/phpunit/StorageWriterTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageWriter\Tests;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageWriter\Component;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+
+class StorageWriterTest extends TestCase
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function setUp() : void
+    {
+        parent::setUp();
+        if (empty(getenv('KBC_TEST_TOKEN')) || empty(getenv('KBC_TEST_URL'))) {
+            throw new \Exception("KBC_TEST_TOKEN or KBC_TEST_URL is empty");
+        }
+        $this->client = new Client([
+            'token' => getenv('KBC_TEST_TOKEN'),
+            'url' => getenv('KBC_TEST_URL'),
+        ]);
+        $tables = $this->client->listTables('in.c-wr-storage-test');
```

Comment:
testy očekávájí že ten bucket `in.c-wr-storage-test`, mělo by to být minimálně zmíněno v README

Reply:
možná by taky bylo lepší když by ten bucket byl v ENV `KBC_TEST_BUCKET`